downstream/modules/platform/proc-upgrade-installer.adoc

```diff
@@ -64,6 +72,21 @@ node_type=execution
 <1> Specifies a control node that runs project and inventory updates and system jobs, but not regular jobs. Execution capabilities are disabled on these nodes.
 <2> Specifies peer relationships for node-to-node connections in the `[execution_nodes]` group.
 <3> Specifies hop nodes that route traffic to other execution nodes. Hop nodes cannot execute automation.
+. Import or generate a {HubName} API token.
++
+* Import an existing API token with the `automationhub_api_token` flag:
++
+[source,options="nowrap",subs=attributes+]
+----
+automationhub_api_token=<api_token>
+----
++
+* Generate a new API token, and invalidate any existing tokens, by setting the `generate_automationhub_token` flag to `True`:
++
+[source,options="nowrap",subs=attributes+]
+----
+generate_automationhub_token=True
+----
```

Comment:
Does it need to be in caps?

Reply:
Just to keep it consistent with the rest of the doc, which all have it capitalized

Reply:
Coolness. I was actually wondering if booleans were case sensitive but there's a pretty good explanation on stack overflow here: https://stackoverflow.com/questions/47877464/how-exactly-does-ansible-parse-boolean-variables

TL;DR I was just curious if there was a technical reason but `True` is good to go.